examples/05-Q3D/Q2D_Armoured_Cable.py

```diff
@@ -0,0 +1,344 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Initialize radii of further structures incrementally adding thicknesses
+
+filling_radius = 1.4142 * (core_xy_coord + 3 * c_strand_radius + core_xlpe_ins_thickness + 0.5)
+inner_sheath_radius = filling_radius + 0.75
+armour_thickness = 3
+armour_radius = inner_sheath_radius + armour_thickness
+outer_sheath_radius = armour_radius + 2
+
+#################################################################################
+# Initialize armature strand dimensions
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Initialize radii
+
+armour_centre_pos = inner_sheath_radius + armour_thickness / 2.0
+arm_strand_rad = armour_thickness / 2.0 - 0.2
+n_arm_strands = 30
+
+#################################################################################
+# Initialize dictionaries
+# ~~~~~~~~~~~~~~~~~~~~~~~
+# Initialize dictionaries that contain all the definitions for the design
+# variables and output variables.
+
+core_params = {
+    "n_cores": str(cable_n_cores),
+    "n_strands_core": str(core_n_strands),
+    "c_strand_radius": str(c_strand_radius) + 'mm',
+    "c_strand_xy_coord": str(core_xy_coord) + 'mm'
+}
+outer_params = {
+    "filling_radius": str(filling_radius) + 'mm',
+    "inner_sheath_radius": str(inner_sheath_radius) + 'mm',
+    "armour_radius": str(armour_radius) + 'mm',
+    "outer_sheath_radius": str(outer_sheath_radius) + 'mm'
+}
+armour_params = {
+    "armour_centre_pos": str(armour_centre_pos) + 'mm',
+    "arm_strand_rad": str(arm_strand_rad) + 'mm',
+    "n_arm_strands": str(n_arm_strands)
+}
+
+#################################################################################
+# Initialize Q2D
+# ~~~~~~~~~~~~~~
+# Initialize Q2D, providing the version, path to the project, and the design
+# name and type.
+
+desktop_version = "2023.2"
+project_name = 'Q2D_ArmouredCableExample'
+q2d_design_name = '2D_Extractor_Cable'
+setup_name = "MySetupAuto"
+sweep_name = "sweep1"
+twinb_design_name = 'CableSystem'
+q2d = pyaedt.Q2d(projectname=project_name, designname=q2d_design_name, specified_version=desktop_version)
+
+##########################################################
+# Define variables from dictionaries
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Define design variables from the created dictionaries.
+
+for k, v in core_params.items():
+    q2d[k] = v
+for k, v in outer_params.items():
+    q2d[k] = v
+for k, v in armour_params.items():
+    q2d[k] = v
+
+##########################################################
+# Create object to access 2D modeler
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Create the object ``mod2D`` to access the 2D modeler easily.
+
+mod2D = q2d.modeler
+mod2D.delete()
+mod2D.model_units = "mm"
+
+#################################################################################
+# Initialize required material properties
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Cable insulators require specific materials not included in the Sys Library
```

Comment:
Intro sentence should have concluding punctuation, but I'm not quite sure what the next line is part of the intro sentence.

Reply:
We'll keep this intro sentence, thanks!